lib/services/local/access_list.go

```diff
@@ -165,6 +166,22 @@ func (a *AccessListService) ListAccessLists(ctx context.Context, pageSize int, n
 	return a.service.ListResources(ctx, pageSize, nextToken)
 }
 
+// ListAccessListsWithFilter returns a filtered and sorted paginated list of access lists.
+func (a *AccessListService) ListAccessListsWithFilter(ctx context.Context, pageSize int, nextToken string, search string, sortBy *types.SortBy) ([]*accesslist.AccessList, string, error) {
+	if sortBy != nil && (sortBy.Field != "name" || sortBy.IsDesc != false) {
+		return nil, "", trace.BadParameter("unsupported sort, only name:asc is supported, but got %q (desc = %t)", sortBy.Field, sortBy.IsDesc)
+	}
```

Comment:
Could you please add a short comment here? Right now i know taht this is just fallback logic with limited scope and that we rely on cache capabilities for filtering,  but I’ll probably forget the context after a week and end up spending time trying to figure out "why"